docs/accessibility.md

```diff
@@ -0,0 +1,87 @@
+For us, accessible means:
+
+- There are text alternatives for non-text content (videos, images, audio, interactive elements like charts).
+- Content can be accessed in different ways, from a computer, phone, or assistive technology, without losing meaning.
+- All functionality is available using a keyboard.
+- Users have enough time to read and react to content.
+- Content is clearly outlined and well-labeled, so that users always know where they are and how to find what they want.
+- Content is always localized, and the language of the page and sections are clearly labelled.
+- We use labels and accessible notifications to let users know about errors and how to correct them.
+- [We don't cause seizures](https://www.w3.org/WAI/WCAG20/quickref/#seizure) with animations or flashing notifications.
+- We do all this within markup standards and best practices to ensure future compatibility.
+
+— (paraphrased from the [WCAG 2.0 Overview](https://www.w3.org/WAI/WCAG20/glance/Overview))
+
+For a detailed list of requirements, you can go to [the WCAG 2.0 customized quickref](https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_customize&levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl). There are 3 levels of criteria for each guideline, A (lowest), AA, and AAA (highest). For Calypso, we aim for WCAG 2.0 Level AA (which also includes level A). This is generally an agreed-upon standard for compliance with equal access laws.
+
+## Automated Testing/Checking
+
+Our ESLint rules include some basic accessibility checks using the [jsx-a11y plugin](https://github.com/evcohen/eslint-plugin-jsx-a11y).
+
+## Resources
+
+A collection of resources that will introduce you to the concept of accessibility and some of the standards behind it.
+
+* [WebAIM](http://webaim.org/): WebAIM has provided comprehensive web accessibility solutions since 1999, and is one of the leading providers of web accessibility expertise internationally. WebAIM is a non-profit organization within the [Center for Persons with Disabilities](http://www.cpd.usu.edu/) at [Utah State University](http://www.usu.edu/). The site brings together a huge collection of information about web accessibility. Most of it is up to date and evergreen enough to help you. The [organization's blog](http://webaim.org/blog/) tackles many modern-day, evolving topics related to web accessibility.
+* [W3C’s Web Accessibility Initiative](http://www.w3.org/WAI/): The World Wide Web Consortium's Web Accessibility Initiative provides strategies, guidelines and resources to make the Web accessible to people with disabilities.
+* [The Six Simplest Accessibility Tests Anyone Can Do](http://www.karlgroves.com/2013/09/05/the-6-simplest-web-accessibility-tests-anyone-can-do/): A handful of simple things you can do to think about and implement accessibility with little effort as you work on projects.
+* [Intro to Cognitive Accessibility](jkg3.com/Journal/cognitive-accessibility-101-part-1-what-is-cognitive-accessibility)
```

Comment:
In the rich text preview, this was redirecting to github, might be safer to use the add the http:// protocol prefix here.

Reply:
Good catch, fixed 👍